### DISCUSSION TITLE
VSCode Raw Go IDE support

### DISCUSSION THREAD

**Discussion by alexander-bruun** [a-h/templ#855] [Q&A]
Is there a way to enable Raw Go support in the VSCode extension to not cause error highligts on things like this?

```templ
{{ totalPages := int(math.Ceil(float64(totalCount) / 9)) }}
```

**Comment by a-h** [comment]
I think it's time to let raw go leave experimental mode. What do you think @joerdav

**Reply by joerdav** [reply]
Yeah agree on this, I think we've proven the experiment, though I do reckon we should have an experiment config option in all the plugins, I can look into adding that.
